tests/utils/inputparams/testparams.go

```diff
@@ -42,8 +43,13 @@ func GetVolumeName() string {
 }
 
 // GetVolumeNameWithTimeStamp prepares unique volume name by appending current time-stamp value
-func GetVolumeNameWithTimeStamp(volName string) string {
-	return volName + "_" + strconv.FormatInt(time.Now().Unix(), 10)
+func GetVolumeNameWithTimeStamp() string {
+	return "volume_" + strconv.FormatInt(time.Now().Unix(), 10)
+}
+
+// GetContainerNameWithTimeStamp prepares unique container name by appending current time-stamp value
+func GetContainerNameWithTimeStamp() string {
+	return "busybox_" + strconv.FormatInt(time.Now().Unix(), 10)
```

Comment:
I would suggest avoid using `busybox_` ... if would be confusing while creating container of `ubuntu` base image. Let' use generic name like `container_`